### PR TITLE
fix docstring

### PR DESCRIPTION
**Pull request by LearningPawn**
### PR types

Others

### PR changes

Docs

### Describe

1. 去除 python/paddle/fluid/dygraph/nn.py 中 BatchNorm 类说明中示例代码里 numpy 的使用，为参数 data_layout 说明中 "NCHW" 里的各字母含义做了具体说明；
2. 为 python/paddle/nn/functional/norm.py 中 batch_norm 函数中参数 data_format 说明中的 "NCDHW" 里的各字母含义做了具体说明；
3. 为 python/paddle/nn/layer/norm.py 中 BatchNorm、BatchNorm1D、BatchNorm2D 以及 BatchNorm3D 类中参数 data_format 说明中的 "NCDHW" 里的各字母含义做了具体说明；
4. 修正 python/paddle/nn/layer/loss.py 中 BCELoss 类的 Shapes 说明格式错误问题，以及 BCEWithLogitsLoss 类的 Shapes 说明格式错误问题和 Examples 出错问题；
5. 修改 python/paddle/nn/functional/loss.py 中 binary_cross_entropy 函数以及 binary_cross_entropy_with_logits 函数 Returns 说明的描述，使它们与其他函数说明相似；
6. 将 python/paddle/nn/decode.py 中 BeamSearchDecoder 类 `__init__` 方法 docstring 里的参数说明提前到类说明中去。

中文文档：https://github.com/PaddlePaddle/docs/pull/5514

### CODE REVIEW

File: python/paddle/nn/functional/norm.py [hunk 1]
```diff
@@ -153,7 +153,7 @@ def batch_norm(
         epsilon(float, optional): The small value added to the variance to prevent division by zero. Default: 1e-5.
         momentum(float, optional): The value used for the moving_mean and moving_var computation. Default: 0.9.
         training(bool, optional): True means train mode which compute by batch data and track global mean and var during train period. False means inference mode which compute by global mean and var which calculated by train period. Default False.
```
Comment (Ligoml) [inline python/paddle/nn/functional/norm.py:154]: training 和 momentum 的顺序需要换一下

**Review by Ligoml — COMMENTED** [review]
good job！有几处小问题反馈在表单中了

**Comment by LearningPawn** [comment]
@Ligoml 已经根据您的意见进行了 docstring 的修改，感谢您的 Review 和指导！

**Review by Ligoml — APPROVED** [review]
good job！LGTM for docs
